### PR TITLE
feat: group chat members

### PR DESCRIPTION
**Pull request by imGok**
image
image
image

#241

### CODE REVIEW

File: assets/l10n/intl_fr.arb [hunk 1]
```diff
@@ -519,7 +519,7 @@
       "error": {}
     }
   },
-  "countParticipants": "{count} participant(s)",
+  "countParticipants": "{count} membres",
```
Comment (sherlockvn) [inline assets/l10n/intl_fr.arb:522]: members

File: assets/l10n/intl_en.arb [hunk 2]
```diff
@@ -604,7 +604,7 @@
             "error": {}
         }
     },
-    "countParticipants": "{count} participants",
+    "countParticipants": "{count} members",
```
Comment (sherlockvn) [inline assets/l10n/intl_en.arb:607]: I think change the name to countMembers

File: lib/utils/room_status_extension.dart [hunk 3]
```diff
@@ -43,8 +43,9 @@ extension RoomStatusExtension on Room {
       }
       return L10n.of(context)!.onlineLongTimeAgo;
     }
-    return L10n.of(context)!
-        .countParticipants(summary.mJoinedMemberCount.toString());
+
+    return L10n.of(context)!.countParticipants(
+      ((summary.mInvitedMemberCount ?? 0) + (summary.mJoinedMemberCount ?? 0)).toString());
```
Comment (sherlockvn) [inline lib/utils/room_status_extension.dart:48]: when you invite new member, did this count is updated ?

File: assets/l10n/intl_fr.arb (hunk 1 above)
Comment (imGok) [reply assets/l10n/intl_fr.arb:522]: no it's french

File: lib/utils/room_status_extension.dart (hunk 3 above)
Comment (imGok) [reply lib/utils/room_status_extension.dart:48]: yes
yes

**Review by hoangdat — APPROVED** [review]

**Review by sherlockvn — APPROVED** [review]

**Pull request merged**